app/models/queries/filter.rb

```diff
@@ -79,7 +79,7 @@ class Queries::Filter
 
   attr_accessor :field, *@@filter_params
 
-  validates_presence_of :field
+  validates_presence_of :field, :operator
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.